lib/Test/Most.pm

```diff
@@ -278,15 +278,17 @@ sub set_failure_handler {
 
 {
     no warnings 'redefine';
+    my $orig_destroy = Test::Builder->can('DESTROY');
 
     # we need this because if the failure is on the final test, we won't have
     # a subsequent test triggering the behavior.
-    sub Test::Builder::DESTROY {
+    *Test::Builder::DESTROY = sub {
```

Comment:
This line shouldn't need to be changed (the latter just sets the subname to `__ANON__`)

Reply:
Or I guess it's depending on the redefinition not happening until runtime

Reply:
right.